sdk/tests/deqp/framework/referencerenderer/rrRenderer.js

```diff
@@ -861,23 +869,32 @@ rrRenderer.triangleRasterizer.prototype.rasterize = function() {
         var barycentric1 = deMath.divide(b1, bSum);
         var barycentric2 = deMath.subtract(deMath.subtract([1, 1, 1, 1], barycentric0), barycentric1);
 
+        // In native dEQP, after rasterization, the pixel (x0, y0) actually represents four pixels:
+        // (x0, y0), (x0 + 1, y0), (x0, y0 + 1) and (x0 + 1, y0 + 1).
```

Comment:
Right, and this caused texture sampling incorrect when texture size is small.

I had a comment somewhere about this, but now I can't find it. :(